src/pyradiosky/skymodel.py

```diff
@@ -3395,6 +3402,10 @@ def read_skyh5(
         ----------
         filename : str
             Path and name of the skyh5 file to read.
+        skip_params : str or list of str or bool
+            A list of optional parameters to skip on read. If set to True, skip
+            all truly optional parameters. The default is False, so by default all
+            optional parameters will be read.
```

Comment:
I think I would add to this that optional parameters that don't exist in the file are already dealt with correctly (i.e. skipped), so that `skip_params` is only for forcing the skipping of parameters that *do* exist in the file but are unnecessary to read for a user's specific application.

Reply:
done.